.gitignore

```diff
@@ -34,3 +34,5 @@ out/
 
 prod.env
 minio_data/
+/doc/api/generated-snippets/
+/doc/javadoc/
```

Comment:
not really necessary, as one should not run the javadoc task manually.
but not really important

Reply:
I just want to prevent accidental commits.